bigchaindb/consensus.py

```diff
@@ -1,9 +1,6 @@
-import copy
 from abc import ABCMeta, abstractmethod
 
-import bigchaindb.exceptions as exceptions
-from bigchaindb import util
-from bigchaindb import crypto
+from . import crypto, exceptions, util
```

Comment:
Lets not use relative imports or if we decide to switch to relative imports we should apply it throughout the entire code